### PR TITLE
Add grafana-dev.net to public suffix list

### PR DESCRIPTION
**Pull request by tolzhabayev**
Adding grafana-dev.net

# Public Suffix List (PSL) Submission

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (`make test`)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the `_psl` TXT record in place in the respective zone(s).

__Submitter affirms the following:__ 

 * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

 * [x] This request was _not_ submitted with the objective of working around other third-party limits.

 * [x] The submitter acknowledges that it is their responsibility to maintain the domains within their section. This includes removing names which are no longer used, retaining the _psl DNS entry, and responding to e-mails to the supplied address. Failure to maintain entries may result in removal of individual entries or the entire section.

 * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms to them.
 * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting.

---

For PRIVATE section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*. I could break my organization's website cookies and cause other issues, and the rollback timing is acceptable. *Proceed anyways*.
---

## Description of Organization

Grafana Labs is behind leading open source projects Grafana and Loki, and the creator of the first open & composable observability platform. We also offer a fully managed solution we call "Grafana Cloud" - without the overhead of building and maintaining your own stack. 

Submitter: Engineering Manager at Grafana Labs

**Organization Website:**

https://grafana.com/

## Reason for PSL Inclusion

Grafana Cloud cookie security: Make our cloud offering more secure - prevent super cookies and ensuring sites on subdomains stay separate.

**Number of users this request is being made to serve:**

- The affected domains listed in this PR will affect approximately ~200 instances which is about 2000 users, once this is successful we will add more domains to it. Once we add all domains we will have more than 5000 customers and tens of thousands of users that would benefit from the PSL inclusion.

## DNS Verification

```
dig +short TXT _psl.grafana-dev.net
"https://github.com/publicsuffix/list/pull/2188"
```

## Results of Syntax Checker (`make test`)

All tests are passing

```
cd linter;                                \
  ./pslint_selftest.sh;                     \
  ./pslint.py ../public_suffix_list.dat;
test_NFKC: OK
test_allowedchars: OK
test_dots: OK
test_duplicate: OK
test_exception: OK
test_punycode: OK
test_section1: OK
test_section2: OK
test_section3: OK
test_section4: OK
test_spaces: OK
test_wildcard: OK
test -d libpsl || git clone --depth=1 https://github.com/rockdaboot/libpsl;   \
  cd libpsl;                                                                    \
  git pull;                                                                     \
  echo "EXTRA_DIST =" >  gtk-doc.make;                                          \
  echo "CLEANFILES =" >> gtk-doc.make;                                          \
  autoreconf --install --force --symlink;
Cloning into 'libpsl'...
Already up to date.
autopoint: using AM_GNU_GETTEXT_REQUIRE_VERSION instead of AM_GNU_GETTEXT_VERSION
Copying file ABOUT-NLS
Creating directory build-aux
Copying file build-aux/config.rpath
Copying file m4/gettext.m4
Copying file m4/host-cpu-c-abi.m4
Copying file m4/iconv.m4
Copying file m4/intlmacosx.m4
Copying file m4/lib-ld.m4
Copying file m4/lib-link.m4
Copying file m4/lib-prefix.m4
Copying file m4/nls.m4
Copying file m4/po.m4
Copying file m4/progtest.m4
Copying file po/Makefile.in.in
Copying file po/Makevars.template
Copying file po/Rules-quot
Copying file po/boldquot.sed
Copying file po/en@boldquot.header
Copying file po/en@quot.header
Copying file po/insert-header.sin
Copying file po/quot.sed
Copying file po/remove-potcdate.sin
  CCLD     libpsl_icu_load_dafsa_fuzzer
  CCLD     libpsl_icu_load_fuzzer
  CCLD     libpsl_icu_fuzzer
PASS: libpsl_icu_load_dafsa_fuzzer
PASS: libpsl_icu_fuzzer
PASS: libpsl_icu_load_fuzzer
============================================================================
Testsuite summary for libpsl 0.21.[5](https://github.com/publicsuffix/list/actions/runs/11105165902/job/32168772207?pr=2188#step:6:6)
============================================================================
# TOTAL: 3
# PASS:  3
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in tests
  CC       test-is-public.o
  CC       common.o
  CC       test-is-cookie-domain-acceptable.o
  CC       test-is-public-all.o
  CC       test-is-public-builtin.o
  CC       test-registrable-domain.o
  CCLD     test-is-public
  CCLD     test-is-cookie-domain-acceptable
  CCLD     test-is-public-all
  CCLD     test-is-public-builtin
  CCLD     test-registrable-domain
PASS: test-is-public-builtin
PASS: test-is-public
PASS: test-is-cookie-domain-acceptable
PASS: test-registrable-domain
PASS: test-is-public-all
============================================================================
Testsuite summary for libpsl 0.[21](https://github.com/publicsuffix/list/actions/runs/11105165902/job/32168772207?pr=2188#step:6:22).5
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in msvc
```

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -13663,6 +13663,10 @@ london.cloudapps.digital
 // Submitted by <domeinnaam@minaz.nl>
 gov.nl
 
+// Grafana Labs https://grafana.com/
+// Submitted by <timur.olzhabayev@grafana.com>
```
Comment (groundcat) [inline public_suffix_list.dat:13667]: Please use a non-personal e-mail.

File: public_suffix_list.dat [hunk 2]
```diff
@@ -13663,6 +13663,10 @@ london.cloudapps.digital
 // Submitted by <domeinnaam@minaz.nl>
 gov.nl
 
+// Grafana Labs https://grafana.com/
```
Comment (groundcat) [inline public_suffix_list.dat:13666]: Please add a colon

**Comment by groundcat** [comment]
> Number of users this request is being made to serve:
> The affected domains listed in this PR will affect approximately ~200 instances, once this is successful we will move on with further, more used, domains.

Could you provide more details on the specific use case of this domain? For instance, how many end users are expected to benefit from the PSL inclusion? It would be helpful to know the average number of users per instance. Additionally, if you have any other relevant data or evidence, we will consider that as well. Please include a concise description of the core focus of the submitting organization, particularly in connection to this request.

Please also note that, according to the [guidelines](https://github.com/publicsuffix/list/wiki/Guidelines#validation-and-non-acceptance-factors):

> Projects that are smaller in scale or are temporary or seasonal in nature will likely be declined. Examples of this might be private-use, sandbox, test, lab, beta, or other exploratory nature changes or requets. It should be expected that despite whatever site or service referred a requestor to seek addition of their domain(s) to the list, projects not serving more then thousands of users are quite likely to be declined.

File: public_suffix_list.dat (hunk 2 above)
Comment (tolzhabayev) [reply public_suffix_list.dat]: thanks, added https://github.com/publicsuffix/list/pull/2188/commits/2ed0b3ed136957381a7d75790f1729e9185cc096

File: public_suffix_list.dat (hunk 1 above)
Comment (tolzhabayev) [reply public_suffix_list.dat]: Switched to non-personal e-mail and display name https://github.com/publicsuffix/list/pull/2188/commits/2ed0b3ed136957381a7d75790f1729e9185cc096

**Comment by groundcat** [comment]
- [ ]  Expiration (Note: Must STAY >2y at all times)
  - [ ]  grafana-dev.net expires 2026-03-25
- [x]  DNS _psl entries (Note: Must STAY in place)
  - `"https://github.com/publicsuffix/list/pull/2188"` OK
- [x]  Tests pass
- [x]  Sorting
- [x]  Reasoning/Organization description
- [x]  Non-personal email address

**Comment by groundcat** [comment]
> * grafana-dev.net expires 2026-03-25

@tolzhabayev The domain grafana-dev.net expires on 2026-03-25 according to the WHOIS record. Please renew it to ensure the expiration date is more than two years from today, as it is required to maintain this two-year buffer **continuously in future years** to keep it in the PSL. Kindly review the guidelines for further details.

**Comment by wdhdev** [comment]
@tolzhabayev Is this PR still needed? We have not received a response regarding the expiry of the domain not being >2y.

**Comment by tolzhabayev** [comment]
I am looking into the expiry at the moment, will respond soon

**Comment by simon-friedberger** [comment]
@tolzhabayev Not all registrars allow a 2 year validity. Some just auto-renew and exceptions can be made for that.

**Comment by tolzhabayev** [comment]
@wdhdev @simon-friedberger we have renewed the domain past 2 years from now. thanks for your patience.

**Pull request merged**